### ISSUE TITLE
MacOS 11.6.4

### ISSUE THREAD

**Issue by villa-one** [DavidMcLaughlin208/PixelSorting#4]
Hello
I tried to launch the app on MacOS 11.6.4 but it crashes. Is there any way to make it work? Thanks

**Comment by DavidMcLaughlin208** [comment]
Yes sorry about that. At the top of the release it says the Mac build doesn't work right now but its not very big.

I have spent 20+ trying to get it to work and now the final part is including the OpenCV library in the app as a static library and I wasn't able to do it before starting my new job. I haven't been able to take a look at it again as of yet. I will try to find time soon but I cannot make any promises when it will be completed.

**Comment by NeuroNoNeuro** [comment]
> Yes sorry about that. At the top of the release it says the Mac build doesn't work right now but its not very big.
> 
> I have spent 20+ trying to get it to work and now the final part is including the OpenCV library in the app as a static library and I wasn't able to do it before starting my new job. I haven't been able to take a look at it again as of yet. I will try to find time soon but I cannot make any promises when it will be completed.

Any luck yet?  Same issue, and I've tried many of the techniques for opening 'damaged' files or uncooperative files in general.

**Comment by matteP6** [comment]
Same issue as @NeuroNoNeuro in ventura.

**Comment by BuildBackBuehler** [comment]
Hmm. Well, I'm using it all the way out on Sonoma and can't open due to "damages". However, when I run the Unix file, pre-haberdashery, it just states that PS cannot find '/usr/lib/libopencv_gapi.4.5.dylib'. 

As far as I can tell from ``sudo find / -name 'libopencv_gapi'.*``, it isn't on my MBP. But I also have an M1, ARM, so its a losing battle I suppose 😭

It still wouldn't work after running some xattr and unquarantining and codesign signing. But I do have SIP on atm, might just have to take the 5mins to disable it to see if it somehow helps make this puppy fly. 

But I think relying on Rosetta is a mistake. Better off trying w/ Pipenv virtual env. – figure I may be missing some package deps that brew's gcc/opencv are missing.

Note, I'm a self-taught pleb, so my apologies @ reader, if its unuseful, but I'll update if I get it working.
